config/routes.js

```diff
@@ -95,6 +97,7 @@ module.exports.routes = {
   'POST   /noo/tag/:communityId/:tagName/follow':         'TagController.follow',
   'GET    /noo/tags/:communityId/followed':               'TagController.findFollowed',
   'GET    /noo/tags/:communityId/leftnav':                'TagController.findForLeftNav',
+  'GET    /noo/tag-in-all-communities/:tagName/posts':    'PostController.findForTagInAllCommunities',
```

Comment:
i should have caught this in the previous review. the `/noo/tag(s)/:communityId...` routes should be `/noo/community/:communityId/tags...` in keeping with the convention we've established where an ID in a URL is for the object named immediately preceding it. then this route becomes simply `/noo/tag/:tagName/posts`. please refactor.